pywinauto/base_application.py

```diff
@@ -594,13 +595,19 @@ def _ctrl_identifiers(self):
 
         return control_name_map
 
-    def print_control_identifiers(self, depth=None, filename=None):
+    def print_control_identifiers(self, depth=30, max_width=5, filename=None):
```

Comment:
Please set default `max_width=10` or maybe even 15-20. Also please check how much time it takes with `depth=30` for infinite tree case.

Reply:
Default `max_width` and `depth` changed to 10.
Dolphin 5.0-9281 takes ~180s with `depth=30` and ~15-20s with `depth=10`